tests/tacacs/test_ro_disk.py

```diff
@@ -40,6 +40,12 @@ def simulate_ro(duthost):
     logger.info("Disk turned to RO state; pause for 30s before attempting to ssh")
     assert wait_until(30, 2, 0, check_disk_ro, duthost), "disk not in ro state"
 
+    # Wait for disk remount finish
+    # Can't check remount finish by command here:
+    #  duthost.shell will failed with "no usable temporary directory found" error
```

Comment:
>"no usable temporary directory found" [](http://example.com/codeflow?start=38&length=37)

Is it possible to wait this error happen for "remount finish"?

Why "use ssh_remote_run before remount finish will cause UT failed"? Is it fixable?

Reply:
For the "no usable temporary directory found" issue, I retry 30seconds and duthost.shell() still can't run any command, it's because the RO disk fix script does not mount any tmp directory.

And for "use ssh_remote_run before remount finish will cause UT failed", I check some failed UT syslog, they have exactly same pattern in syslog: try login to device during remount happen. I still not found the reason, but the remount trigger by the UT:
duthost.shell("echo u > /proc/sysrq-trigger")

Reply:
Improve the code to wait 10seconds, add TODO for improve with rsyslog.